### PR TITLE
Add Delete Client/Property to Implementation section in Developer Guide

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**
Not the final version yet

### CODE REVIEW

**Review by wilsonngja — COMMENTED** [review]
Overall the documentation of DG looks good. However, some of the text inside the class diagram can be quite difficult to see. Can consider increasing the font size slightly and change the width of the box so that the text wont be brought over to the next line. Great work.

**Review by OVReader — APPROVED** [review]
Great work on the DG draft; LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -81,7 +81,46 @@ operations, the text files will be updated when `add`, `delete`, `pair` or `unpa
...
+
+The following *sequence diagram* shows how the **delete client** operation works, showcasing the
+```ClientList#deleteClient()``` method.
+
+![Delete Client Sequence Diagram](diagrams/DeleteClientSD.png)
+
+The following *sequence diagram* shows how the **delete property** operation works, showcasing the
+```PropertyList#deleteClient()``` method.
+
+![Delete Property Sequence Diagram](diagrams/DeletePropertySD.png)
```
Comment (ngdeqi) [inline docs/DeveloperGuide.md:123]: Do ensure that your lifeline extends continuously downwards for your sequence diagrams.

Comment (ngdeqi) [inline docs/DeveloperGuide.md:123]: Also, the return arrows for sequence diagrams should be dashed

**Pull request merged**